packages/admin-app/src/components/ElectionSetupWizard/ElectionSetupWizard.tsx

```diff
@@ -46,10 +49,14 @@ export const ElectionSetupWizard: React.FC = () => {
         };
         setRequest(newRequest);
     };
-
-    const handleSubmit = () => {
+    const handleUploadManifest = (manifestJson: ValidateManifestRequest) => {
+        setManifest(manifestJson);
+    };
+    const handleSubmit = async () => {
+        const v1Client = ClientFactory.GetV1Client();
```

Comment:
I believe we should handle this with ReactContext. Reduce getting the client over and over.

Reply:
This refactor will affect a lot of files and cause fairly substantial merge conflicts with my downstream work.  If you don't mind, I'll implement this in the PR I'm about to submit for #124.